src/app/data-planes/views/DataPlanePolicySummaryView.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <RouteView
+    v-slot="{ route, t }"
+    name="data-plane-policy-summary-view"
+    :params="{
+      mesh: '',
+      policyPath: '',
+      policy: '',
+      codeSearch: '',
+      codeFilter: false,
+      codeRegExp: false,
+    }"
+  >
+    <DataSource
+      v-slot="{ data, error }: PolicySource"
+      :src="`/meshes/${route.params.mesh}/policy-path/${route.params.policyPath}/policy/${route.params.policy}`"
+    >
+      <AppView>
+        <template #title>
+          <PolicySummaryTitle
+            :title="t('policies.routes.item.title', { name: route.params.policy })"
+            :link-to="{
+              name: 'policy-detail-view',
+              params: {
+                mesh: route.params.mesh,
+                policyPath: route.params.policyPath,
+                policy: route.params.policy,
+              },
+            }"
+          />
+        </template>
+        <DataLoader
+          :data="[data]"
+          :errors="[error]"
+        >
+          <PolicySummary
+            v-if="data"
+            :policy="data"
+          >
```

Comment:
I can see there is a little more reason to add this as a reusable component and use it twice, and I like that you kept the `route` access outside the component by using a slot 👍 . Feel free to resolve this comment just wanted to point out I liked this.